_docs/2.0.0/usage/menas-conformance-rules.md

```diff
@@ -14,46 +14,50 @@ categories:
 
 Every column also has one or more input columns.
 
 **Important** - We never override a column. Each rule produces a new column.
 
 ## Casting Conformance Rule
 
-Allows users to cast a specific column to another type. This conformance rules provides a selection of other types.
+Casting conformance rule Allows users to cast a specific column to another type. This conformance rule provides a selection of other types.
 
 ## Concatenation Conformance Rule
 
-Concatenation rule concatenates multiple input columns together into a single column. Columns are first transformed into their string representation.
+Concatenation conformance rule concatenates two or more input columns together into a single column. Columns are first transformed into their string representation before the actual concatenation.
 
 ## Drop Conformance Rule
 
-Drop Conformance rule remove a column from the DataFrame.
+Drop conformance rule removes a column from the output data.
+
+## FillNulls Conformance Rule
+
+FillNulls conformance rule takes a column and replaces all the nulls with a literal provided by the user.
 
 ## Literal Conformance Rule
 
-Literal rule add a column with a string literal provided by the user.
+Literal conformance rule adds a column with a string literal provided by the user.
 
 ## Mapping Conformance Rule
 
-To use a mapping conformance rule, user first needs to define a Mapping Table in menas. Mapping Tables have the same properties and rules around them as Datasets, plus it is expected of them to be in a parquet format.
+To use a mapping conformance rule, the user first needs to define a Mapping Table in Menas. Mapping Tables have the same properties and rules around them as Datasets, and it is expected of them to be in a parquet format.
 
-When defining a mapping conformance rule, users first needs to pick a correct Mapping Table and correct version. Then there is a question if the data can have Null values in join conditions. This means if the join that will be executed should be null safe or not.
+When defining a mapping conformance rule, users first need to pick a correct Mapping Table and correct version. Then there is a question if the data can have Null values in join conditions. This means if the join that will be executed should be null safe or not.
 
-Then the join conditions. A join condition condition conveys a relationship between the Dataset and MappingTable. The join condition specifies how the rows from one table will be combined with the rows of another table. This is based on the equality of the rows.
+Then the join conditions. A join condition conveys a relationship between the Dataset and MappingTable. The join condition specifies how the rows from one table will be combined with the rows of the other table. This is based on the equality of the rows.
```

Comment:
**Comment:** Can't help but find the single sentence _"Then the join conditions."_ at the beginning of the paragraph not fitting. 🤓